lib/details/widgets/body_details_screen.dart

```diff
@@ -54,34 +55,36 @@ class BodyDetailsScreen extends HookConsumerWidget {
               const ChangeDaysButtons(),
               const Divider(thickness: 1),
               RowInfos(
-                title: 'Preço nos últimos $days dias',
+                title:
+                    '${CoreString.of(context)!.pricesfive} $days ${CoreString.of(context)!.days}',
                 info: FormatCurrency.format(
                   data.price.reversed.elementAt(days).last.toDouble(),
                 ),
               ),
               const Divider(thickness: 1),
               RowInfos(
-                  title: 'Variação em $days dias',
+                  title:
+                      '${CoreString.of(context)!.variationfive} $days ${CoreString.of(context)!.days}',
```

Comment:
Olhando o nome dá a entender que ele só se aplica quando a variação do gráfico estiver em 5 dias, como a coluna de informações se adapta quando o range de tempo do gráfico for alterado?

Reply:
Vou mudar o nome pois é confuso mesmo, eu fiz uma tradução para dias e uma para variação em, então fiz uma interpolação com o número do botão. Vou mudar o nome para fazer sentido.